### ISSUE TITLE
Usage with #[tokio::test]

### ISSUE THREAD

**Issue by mkpankov** [frondeus/test-case#36]
Currently it doesn't seem to work:
```rust
    #[tokio::test]
    async fn test_case(name: &str) {
        
    }
```
Output:
```
error: the test function cannot accept arguments
   --> iml-services/iml-devices/src/db.rs:578:24
    |
578 |     async fn test_case(name: &str) {
    |                        ^^^^^^^^^^
```

Is there a way to provide a `test_case::test_case_tokio` macro to support async tests?

**Comment by frondeus** [comment]
That'd be cool... However I'd like to hide it under feature gate.

**Comment by luke-biel** [comment]
It is fixed now via #48

**Comment by luke-biel** [comment]
It is fixed now via #48
